docs/user-guide/src/bmo/bmo.md

```diff
@@ -0,0 +1,44 @@
+<h1>Bare Metal Operator</h1>
+
+The Bare Metal Operator (BMO) is a custom Kubernetes controller that reconciles BareMetalHost resources using information provided by Ironic and the Cluster API Provider Metal3 (CAPM3) controller. A simplified information flow path is shown below:
```

Comment:
Probably worth noting that BMO can be used without  CAPM3.

Reply:
I'd say that main BMO goal is to deploy baremetal hosts as Kubernetes nodes. This is currently performed via Ironic, and the hosts are represented by a BMH resource

Reply:
fixed, thanks.